science/ALPSCore/Portfile

```diff
@@ -5,8 +5,8 @@ PortGroup           cmake 1.0
 PortGroup           github 1.0
 PortGroup           mpi 1.0
 
-github.setup        ALPSCore ALPSCore 0.5.6-alpha3 v
-revision            1
+github.setup        ALPSCore ALPSCore 1.0.0 v
+revision            0
```

Comment:
`revision 0` is the default and is usually left out.

Reply:
Thanks. Will do the next time.